.github/workflows/rust.yml

```diff
@@ -100,16 +100,22 @@ jobs:
       if: ${{ matrix.test && matrix.os != 'windows-latest' }}
       run: |
         set -euxo pipefail
-        for bin in $bins; do ./${{ matrix.output }} binstall --no-confirm $bin; done
-        ./${{ matrix.output }} binstall --manifest-path . --no-confirm cargo-binstall
+        for bin in $bins; do ./${{ matrix.output }} binstall --log-level debug --no-confirm $bin; done
+        for bin in $test_bins; do $HOME/.cargo/bin/$bin --version; done
+        cargo binstall --help >/dev/null
+        ./${{ matrix.output }} binstall --log-level debug --manifest-path . --no-confirm cargo-binstall
+        cargo binstall --help >/dev/null
```

Comment:
This is getting a bit complicated. Can we extract it to a script?

Reply:
Done!